operations/iscsi_sbps/Node_Personalization_Verification.md

```diff
@@ -0,0 +1,256 @@
+
+(`ncn-w#`) Run these checks on worker nodes that are enabled as an iSCSI target in order to verify that it is
+ configured correctly.
+
+### Check SBPS Marshal Agent status
+
+    Check that the SBPS Marshal Agent is running without any errors.
+
+    ```bash
+    systemctl status sbps-marshal
+    ```
+
+    Beginning of example output:
+
+    ```text
+    ● sbps-marshal.service - System service that manages Squashfs images projected via iSCSI for IMS, PE, and other ancillary images similar to PE.
+     Loaded: loaded (/usr/lib/systemd/system/sbps-marshal.service; enabled; preset: disabled)
+     Active: active (running) since Wed 2025-07-23 20:33:15 UTC; 5 days ago
+   Main PID: 2218037 (sbps-marshal)
+      Tasks: 1
+        CPU: 39min 9.487s
+     CGroup: /system.slice/sbps-marshal.service
+             └─2218037 /usr/lib/sbps-marshal/bin/python /usr/lib/sbps-marshal/bin/sbps-marshal
+
+Jul 29 11:30:15 ncn-w002 sbps-marshal[2218037]: agent.py:main:314 INFO 2025-07-29T11:30:15+0000 No sbps-project key value, so image is not mark>
+Jul 29 11:30:15 ncn-w002 sbps-marshal[2218037]: agent.py:main:314 INFO 2025-07-29T11:30:15+0000 No sbps-project key value, so image is not mark>
+Jul 29 11:30:15 ncn-w002 sbps-marshal[2218037]: agent.py:main:314 INFO 2025-07-29T11:30:15+0000 No sbps-project key value, so image is not mark>
+Jul 29 11:30:15 ncn-w002 sbps-marshal[2218037]: agent.py:main:314 INFO 2025-07-29T11:30:15+0000 No sbps-project key value, so image is not mark>
+Jul 29 11:30:15 ncn-w002 sbps-marshal[2218037]: agent.py:main:314 INFO 2025-07-29T11:30:15+0000 No sbps-project key value, so image is not mark>
+Jul 29 11:30:15 ncn-w002 sbps-marshal[2218037]: agent.py:main:314 INFO 2025-07-29T11:30:15+0000 No sbps-project key value, so image is not mark>
+Jul 29 11:30:15 ncn-w002 sbps-marshal[2218037]: agent.py:main:314 INFO 2025-07-29T11:30:15+0000 No sbps-project key value, so image is not mark>
+Jul 29 11:30:15 ncn-w002 sbps-marshal[2218037]: agent.py:main:314 INFO 2025-07-29T11:30:15+0000 No sbps-project key value, so image is not mark>
+Jul 29 11:30:15 ncn-w002 sbps-marshal[2218037]: agent.py:main:314 INFO 2025-07-29T11:30:15+0000 No sbps-project key value, so image is not mark>
+Jul 29 11:30:15 ncn-w002 sbps-marshal[2218037]: agent.py:main:405 INFO 2025-07-29T11:30:15+0000 END SCAN
+    ```
+   
+   We can see in the above example output that `sbps-marshal` agent status is 'Active' and we can check for any errors in the current run
+   by using below command:
+
+   ```bash
+   journalctl -xeu sbps-marshal.service | tac | sed '/START SCAN/q' | grep -i error)
+   ```
+
+   If the above command do not report any errors, then there is no issue else need to look for what error. 
+
+
+### Check `targetcli ls` command output
+
+   Check the `targetcli ls` command output for `fileio` backing store, iSCSI `LUNs`, portals creation.
+
+   i.e Check to see if the `fileio` backing stores exist for `rootfs` images along with corresponding iSCSI `LUNs`.
```

Comment:
This section needs to be elaborated. How does the admin know that the backing stores exist? Or the LUNs?